### PR TITLE
dev-go/gox: Keyword 1.0.1 riscv, #875602

### PR DESCRIPTION
**Pull request by slchris**
Signed-off-by: Chris Su <chris@lesscrowds.org>

### CODE REVIEW

File: dev-go/gox/files/gox-1.0.1-add-riscv-support.patch [hunk 1]
```diff
@@ -0,0 +1,550 @@
+--- a/go.go	2019-04-10 23:02:03.000000000 +0800
```
Comment (jsmolic) [inline dev-go/gox/files/gox-1.0.1-add-riscv-support.patch:1]: Could you add more information about the patch (where it's taken from, author, upstream status)

Comment (slchris) [reply dev-go/gox/files/gox-1.0.1-add-riscv-support.patch:1]: Yap, I'll add more information and force push

File: dev-go/gox/gox-1.0.1.ebuild [hunk 2]
```diff
@@ -7,13 +7,19 @@ inherit go-module
 DESCRIPTION="A dead simple, no frills Go cross compile tool"
 HOMEPAGE="https://github.com/mitchellh/gox"
 SRC_URI="https://github.com/mitchellh/gox/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+SRC_URI+=" https://dev.gentoo.org/~jsmolic/gox-riscv.patch"
 
 LICENSE="MPL-2.0"
 SLOT="0/${PVR}"
 KEYWORDS="amd64 ~arm64 ~ppc64 ~x86"
```
Comment (jsmolic) [inline dev-go/gox/gox-1.0.1.ebuild:14]: No ~riscv keyword?

Comment (jsmolic) [reply dev-go/gox/gox-1.0.1.ebuild:14]: Could you make 2 commits (one that adds the patch and one that adds ~riscv keyword)

Comment (slchris) [reply dev-go/gox/gox-1.0.1.ebuild]: My mistake, I now split it into two commits one with the riscv patch added and one with the riscv keyword added

**Review by jsmolic — APPROVED** [review]

**Pull request closed without merging**